channeldb/channel.go

```diff
@@ -522,6 +527,37 @@ func (c *OpenChannel) MarkAsOpen(openLoc lnwire.ShortChannelID) error {
 	})
 }
 
+// MarkBorked marks the event when the channel as reached an irreconcilable
+// state, such as a channel breach or state desynchronization. Borked channels
+// should never be added to the switch.
+func (c *OpenChannel) MarkBorked(borked bool) error {
```

Comment:
Why's the `borked` bool necessary here? In what situation would the channel ever be "unborked"?

Reply:
was thinking it _could_ be useful in testing, but you're right it isn't necessary in normal operation